### ISSUE TITLE
Windows packaging not working correctly after plugin revert

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/OpenMS#6581]
I think the changes in the NSIS script also need to be reverted. In the beginning I thought it would not matter but by looking at it, it seems like we previously assumed that just every dll and exe is plainly together on the bin folder and it worked.

The problem is now, that I also changed the plugin structure in the install dir to plugin/platformplugins or plugin/other subfolder.

Two options:
1. Revert plugin handling on Windows to "everything in the same folder"
2. Fix qt.conf (somehow contains "/plugins" instead of "plugins" or "./plugins") and  fix NSIS script to install the contents of plugins into bin/plugins instead of bin.

**Comment by timosachsenberg** [comment]
Quick question:  do we use the tool windeployqt and would that help? As I understand, it should create the required folder structure automatically. Or is this not possible because we need to define everything in the cmake scripts

**Comment by poshul** [comment]
I looked into using windeployqt, but for our use case it ended up being more trouble than just manually creating the structure. @jpfeuffer would reverting 12a4f5e6656f257036191170966699c36414fa89 , 8bf7e77012343b4184471013b89d8b38c148657b and d38393251707d4329d0a74492212ac0bfb840fe5 fix the problem?

**Comment by timosachsenberg** [comment]
> I looked into using windeployqt, but for our use case it ended up being more trouble than just manually creating the structure. 
That's interesting! Why is that? What sounded nice about it that it also puts the correct visual studio redistributables in the folder.

**Comment by jpfeuffer** [comment]
To be honest, it would have been nice to have windeployqt years ago. Currently it's advantages are minor.
See my other issue. We already have both vsredist.exe AND the dlls in the installer. We need to choose one.

**Comment by jpfeuffer** [comment]
To be honest, it would have been nice to have windeployqt years ago. Currently it's advantages are minor.
See my other issue. We already have both vsredist.exe AND the dlls in the installer. We need to choose one.

**Comment by jpfeuffer** [comment]
Seems like one does not need qt.conf with Qt 5.14+. It is built "relocatable" by default since then (-> relative paths to plugin dirs).
Not closing, since it might need more tests after #6589 was mergd.

**Comment by poshul** [comment]
As of 2023-03-07 the dlls are bundled in the bin directory in windows.
